source/nodejs/adaptivecards-designer/src/open-sample-dialog.ts

```diff
@@ -61,11 +63,9 @@ class CatalogueItem {
                     if (sender.sampleData) {
                         let template = new ACData.Template(cardPayload);
 
-                        cardPayload = template.expand(
-                            {
-                                $root: JSON.parse(sender.sampleData)
-                            }
-                        );
+                        cardPayload = template.expand({
```

Comment:
Please revert the formatting to what it was before - it was deliberate and clearer.

Reply:
@dclaux  Addressed the review comments.